DefineOptions.cmake

```diff
@@ -20,6 +20,7 @@ option(USE_XMLLINT "Run xmllint to test if darktableconfig.xml is valid" ON)
 option(USE_OPENJPEG "Enable JPEG 2000 support" ON)
 option(USE_WEBP "Enable WebP export support" ON)
 option(USE_AVIF "Enable AVIF support" ON)
+option(USE_HEIF "Enable HEIF/HEIC support" ON)
```

Comment:
I suggest defaulting to OFF due to potential patent infringement risk

Reply:
I disagree and suggest leaving it defaulting to `ON`.
The code in this PR has no part that is covered by patent, the actual "heavy lifting" is done by libheif. The heif support will not be present if darktable is not built with libheif available alleviating all concerns.

IMO - leaving it "ON" is perfecty fine.

Reply:
> The heif support will not be present if darktable is not built with libheif

OTOH, dt will be built and shipped with libheif+libde265 if they were present on the build host (for some reason) without the end user knowing it, and probably not having paid for the HEVC license. Better to explicitly opt-in in these cases, rather than opt-out IMHO.

Reply:
Installing the libheif+libde265 is the step that should be confirmed by user then, it's not a dt issue. I also disagree to having it OFF by default and will close this.